lib/memcached.js

```diff
@@ -112,8 +111,7 @@ Client.config = {
   nMemcached.prototype.__proto__ = require('events').EventEmitter.prototype;
 
   var memcached = nMemcached.prototype
-    , privates = {}
-    , undefined;
```

Comment:
I think the redefining `undefined` part is obsolete in node and other modern runtimes:

```
> undefined = 'foo'
'foo'
> undefined
undefined
```